web_calendar_slot_duration/static/src/js/calendar_common_renderer.esm.js

```diff
@@ -11,7 +11,10 @@ patch(
     {
         get options() {
             const options = this._super(...arguments);
-            if (this.env.searchModel.context.calendar_slot_duration) {
+            if (
+                this.env.searchModel.context.calendar_slot_duration &&
+                this.env.searchModel.context.adapt_view_to_slot_duration
```

Comment:
I don't understand this fix. Could you post some screenshots explaining the problem you're fixing here please? Also, don't you think it would make sense to default to `true` for `adapt_view_to_slot_duration`?

Reply:
Hello @yajo 
The `adapt_view_to_slot_duration` is to solve the way to display the calendar view.
If you check on my screen shot on description, If I put a `slot_duration` like 30m, on the calendar the view will be adapted to display slots 30min by 30min and IMO it's not very "beautiful". 

And yes, I'm agreed with you about the `default` to `true` to not change the standard behavior .

Reply:
OK I see... Well, then it's OK, but could you add some docs in the readme please?